makeflow/src/makeflow.c

```diff
@@ -874,7 +880,14 @@ static void makeflow_run( struct dag *d )
 	struct dag_node *n;
 	batch_job_id_t jobid;
 	struct batch_job_info info;
+        timestamp_t last_time = timestamp_get();
+        timestamp_t start = timestamp_get();
+        int first_report = 1;
 
+        //reporting to catalog
+        if(in_n_mode){
```

Comment:
For someone reading the code, it's not immediately obvious what "N mode" is. Rather than commenting, it might be helpful to give this variable a more descriptive name like `send_reports` or something similar